gto/cli.py

```diff
@@ -366,6 +366,13 @@ def callback_sort(  # pylint: disable=inconsistent-return-statements
     help="Push created tag automatically (experimental)",
 )
 
+option_auto_commit = Option(
+    False,
+    "--auto-push",
```

Comment:
`--auto-commit` I guess?

Reply:
Thanks! fixed in https://github.com/iterative/gto/pull/290/commits/66c5728bb15be0d42b4dba49df8a9ff95fa129e1